### ISSUE TITLE
There should be releases that includes the compiled DOCX and Allfiles content

### ISSUE THREAD

**Issue by seesharprun** [MicrosoftLearning/20533-ImplementingMicrosoftAzureInfrastructureSolutions#166]
Every MCT doesn't have the time or experience to build the labs manually. I recommend that there should be a release every time there is a commit to the master branch with a simple change log.

**Comment by seesharprun** [comment]
Here is an example from the 20532 repository: https://github.com/MicrosoftLearning/20532-DevelopingMicrosoftAzureSolutions/releases

**Comment by grantcarthew** [comment]
Better yet, add CI to the project with [Jenkins](https://jenkins.io/) and github webhook to build every time a PR is merged into master.

**Comment by normandk** [comment]
I totally agree with seesharpun.  The option they have for the 20532 for direct download without the need to build is great.  We need this for 20533 please......

**Comment by polichtm** [comment]
Appreciate the suggestion - we'll look into updating the release mechanism

**Comment by seesharprun** [comment]
Is it possible to leave this issue in an open state until your team reaches a decision.

Deciding to not create GitHub releases is a perfectly valid decision, but closing this issue on GitHub indicates that the issue is resolved and no further action is necessary.

I would recommend adding a tag to the issue indicating that it's "under review".

**Comment by grantcarthew** [comment]
@seesharprun comments apply to all the issues closed recently:
#162, #163, #164, #165, #167, #169

The issue raised by #163 specifically was completely misunderstood.

**Comment by polichtm** [comment]
Gents, 
I'll gladly facilitate your suggestions whenever they are feasible. I'll reopen this issue along with 163 - consider them "under review"
rgds
Marcin

**Comment by grantcarthew** [comment]
Thanks @polichtm 
"Under review" is exactly what these issue tracking systems are for.

Close issues when you decide one of the following:

*   It has been implemented.
*   It is decided the change has no benefit or is outside of the scope of the project.

Probably the most important point is to NOT close issues because time or resources are limited. This is open source development. Anyone who has the time or inclination can and will contribute.

Thanks again.

**Comment by travellingtrainer** [comment]
Could I suggest that the files be pdf and not docx ? That way they can be opened on the VM which doesn't have Word.

**Comment by itbland** [comment]
>Could I suggest that the files be pdf and not docx ? 

I think you'll find that copy/paste out of a PDF will often cause problems for PowerShell (line breaks get inserted)
